y.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+echo "[BUILD] build system" 1>&2
+mkdir build_system/target
```

Comment:
This will error if the target already exists. Maybe use `-p`?

Reply:
Just did. :)